ReadMe.md

```diff
@@ -12,6 +12,7 @@ This is a repository that contains several useful things that other `nblockchain
         * [Use of `-latest` suffix in `runs-on:` GitHubCI tags](scripts/unpinnedGitHubActionsImageVersions.fsx).
         * [Use of asterisk (*) in `PackageReference` items of .NET projects](scripts/unpinnedDotnetPackageVersions.fsx).
         * [Missing the version number in `#r "nuget: ` refs of F# scripts](scripts/unpinnedNugetPackageReferenceVersions.fsx).
+        * [Compile F# scripts](scripts/compileFsharpScripts.fsx).
```

Comment:
@realmarv you placed this as an item part of a list named "use of unpinned versions". This is not about versions. Please pay more attention to detail.